powersimdata/network/usa_tamu/model.py

```diff
@@ -102,11 +103,30 @@ def check_interconnect(interconnect):
     if n > len(set(interconnect)):
         raise ValueError("List of interconnects contains duplicate values")
     if "USA" in interconnect and n > 1:
-        raise ValueError("USA interconnect cannot be paired")
+        raise ValueError("USA cannot be paired")
+    if n == 3:
+        raise ValueError("Use USA instead")
 
     return True
 
 
+def interconnect_to_name(interconnect):
+    """Return name of interconnect or collection of interconnects..
+
+    :param list interconnect: interconnect name(s).
+    """
+    check_interconnect(interconnect)
+    if len(interconnect) == 1:
+        return interconnect[0]
+    elif len(interconnect) == 2:
+        if set(interconnect) == {"Texas", "Western"}:
+            return "Texas_Western"
+        elif set(interconnect) == {"Texas", "Eastern"}:
+            return "Texas_Eastern"
+        elif set(interconnect) == {"Eastern", "Western"}:
+            return "Eastern_Western"
```

Comment:
If we change `Texas_Eastern` to `Eastern_Texas`, this will be aligned with alphabet order, which is equivalent to `'_'.join(sorted(list(set(interconnect))`

Reply:
Do we have any Texas - Eastern scenario in the ***ScenarioList*** already?

Reply:
I don't think so.

Reply:
Done